### DISCUSSION TITLE
How to update? Data lost?

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [lukevella/rallly#319] [Q&A]
How can i upgrade a rallly installation which I did via docker compose up?
Will my previous data be lost during upgrade?

**Comment by lukevella** [comment]
Hi, to get the latest version you will need to pull the latest commit from the repository. If you've cloned the repository all you need to do is run the following in your project root.

```bash
git fetch
git pull
```

Once you've pulled the latest commit you need to rebuild your docker image and restart the service

```bash
docker-compose build
docker-compose up
```

Your data shouldn't be lost because the docker-compose file uses a volume to persist data.
